src/test/compress_tests.cpp

```diff
@@ -2,13 +2,12 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <boost/test/unit_test.hpp>
```

Comment:
third-party libraries are seperated by a new line (like it was before), so please don't change that.

Reply:
Yes, please keep the order of includes the same and only add new ones where necessary.